src/client.rs

```diff
@@ -0,0 +1,41 @@
+use crate::error::Error;
+use prost_types::Any;
+
+pub struct Client(internal::client::DaprClient<tonic::transport::Channel>);
+
+impl Client {
+    /// Connect to a Dapr enabled app.
+    pub fn connect(addr: String) -> Result<Self, Error> {
```

Comment:
A global `connect` functio will be better? Use `dapr::client::connect` instead of `dapr::Client::connect`, and we can change the return type in future.

Reply:
I'm not convinced that a stand alone function is the right way to go. Can you explain why you would prefer that?

And you're right, we should use the async implementation. I will change that.

Reply:
I think I misunderstood you're comment. Are you suggesting using a global function because eventually it could be come async?

Reply:
Agree, `async fn` is a strong point, besides, a global function will make the return type changes more flexible. Please check my comments below.

Reply:
I think it is just a little different habit, this is up to you :)

Generally, I believe `dapr::Client::connect` like a constructor, usually assumes to return a `Client` instance; on the other hand, `dapr::client::connect` more like a factory, it may return someting that behaviors like `Client`. It could be a layered `Client`, like`Tracing<Client>`, `Tracing<LoadBalance<Client>>` etc.

If you prefer to embed all of these features into the `Client`, the two methods are the same.

Reply:
I would imagine a different function would add load balancing and/or tracing support. Clint::new should just give you a plain client.

Reply:
Agree, thanks for your patience :)